content/swagger/resources/products/definitions/product.yaml

```
@@ -134,6 +134,11 @@ properties:
         description: Status of the product regarding the user permissions (only available since the v2.0 in the Enterprise Edition)
         type: string
         enum: ['read_only','draft_in_progress','proposal_waiting_for_approval','working_copy']
+  quality_scores:
+    type: object
+    description: Only available when query parameter "with_quality_scores" is set to "true" (only available in Serenity)
```

As this is a description, you need to actually describe what the object will contain.
```suggestion
    description: Product quality scores for each channel/locale combination (only available in Serenity and when the "with_quality_scores" query parameter is set to "true")
```